.env.virtual-funding

```diff
@@ -17,6 +17,7 @@ RPC_ENDPOINT = http://${GANACHE_HOST}:${GANACHE_PORT}
 ## xstate-wallet
 XSTATE_WALLET_DEPLOYER_ACCOUNT_INDEX = '3'
 LOG_DESTINATION = 'console'
+USE_INDEXED_DB = 'true'
```

Comment:
This might be worth announcing to the dev group as a "breaking change", like we were talking about in today's standup.